doc/sphinx/proof-engine/ltac.rst

```diff
@@ -707,25 +707,25 @@ First tactic to succeed
 In some cases backtracking may be too expensive.
 
 .. tacn:: first [ {*| @ltac_expr } ]
-          first @ltac_expr
+          first @ident
    :name: first; _
 
-   In the first form: for each focused goal, independently apply the first
-   :token:`ltac_expr` that succeeds.
+   In the first form: for each focused goal, apply the first tactic
```

Comment:
You removed "independently", but I think this word was important to clarify that it may not be the same tactic that is selected for each focused goal.